pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -183,6 +183,170 @@ def test_get_indexer():
         idx1.get_indexer(idx2)
 
 
+def test_get_indexer_methods():
+    # https://github.com/pandas-dev/pandas/issues/29896
+    # test getting an indexer for another index with different methods
+    mult_idx_1 = MultiIndex.from_product([[0], [0, 2, 3, 4]])
+    mult_idx_2 = MultiIndex.from_product([[0], [1, 3, 4]])
```

Comment:
example to include at least 2 0-levels, e.g. the backfill / pad operations should not cross the boundaries (doesn't have to be on every one, just some that are relevant)

Reply:
done, thanks for the suggestion!